packages/server/modules/shared/utils/ip.js

```diff
@@ -0,0 +1,15 @@
+const getIpFromRequest = (req) => {
```

Comment:
I feel like the concerns of "resolving the request IP" and "filtering out local network IPs" should be separated out (for ease of re-use), but it doesn't matter that much at this point